packages/core/src/lib/implementation/execute-plugin.unit-test.ts

```diff
@@ -1,30 +1,34 @@
 import { vol } from 'memfs';
-import { describe, expect, it } from 'vitest';
+import { describe, expect, it, vi } from 'vitest';
 import { AuditOutputs, PluginConfig } from '@code-pushup/models';
 import {
   MEMFS_VOLUME,
   MINIMAL_PLUGIN_CONFIG_MOCK,
 } from '@code-pushup/test-utils';
+import { ui } from '@code-pushup/utils';
 import {
   PluginOutputMissingAuditError,
   executePlugin,
   executePlugins,
 } from './execute-plugin';
 
+const nodePluginSlug = MINIMAL_PLUGIN_CONFIG_MOCK.slug;
+
+// eslint-disable-next-line max-lines-per-function
```

Comment:
This is a result of ESLint thinking this is production code. When something unexpected like this happens, instead of disabling ESLint, investigating why this rule is on in the first place should help resolve the typo.
```suggestion
```

Reply:
fixed